### DISCUSSION TITLE
Problems with "Folder with Files"

### DISCUSSION THREAD

**Discussion by Mariahwo** [FlowiseAI/Flowise#1455] [Q&A]
I cannot train AI agents with folders. I have created a folder with training data on my PC with the Windows 11 operating system. To copy the path, I right-clicked on the folder and then clicked on "Copy path". However, I still get an error message saying that it does not recognise my folder. **"Error:** ENOENT: no such file or directory, scandir '"C:\Users\Mariah\Desktop\KI-Lehrer" I have authorisation for the folder. So what could I have done wrong?
![Flowise-Low-code-LLM-apps-builder](https://github.com/FlowiseAI/Flowise/assets/154843450/da074e83-6b28-48b3-aa60-8fe7e5587eb6)

**Comment by HenryHengZJ** [comment]
If you are running through Docker, you'll have to mount the file path to your docker container
